package.json

```diff
@@ -67,9 +67,14 @@
     "build": "node --unhandled-rejections=strict build.mjs",
     "serve": "vite preview"
   },
-  "browserslist": [
-    "supports es6-module",
-    "last 2 versions",
-    "Firefox ESR"
-  ]
+  "browserslist": {
+    "production": [
+      "defaults and supports es6-module"
```

Comment:
We may want to keep Firefox ESR. That is used on the operational consoles at the telescope

Reply:
Yes, `defaults` includes Firefox ESR